ambry-store/src/main/java/com/github/ambry/store/BlobStore.java

```diff
@@ -196,7 +196,13 @@ public BlobStore(ReplicaId replicaId, StoreConfig config, ScheduledExecutorServi
     this.recovery = recovery;
     this.hardDelete = hardDelete;
     this.accountService = accountService;
-    this.replicaStatusDelegates = config.storeReplicaStatusDelegateEnable ? replicaStatusDelegates : null;
+    if (config.storeReplicaStatusDelegateEnable && replicaStatusDelegates != null) {
```

Comment:
Double check it's only print once in the bootup time.

Reply:
printed once per partition but only during server startup